ccmlib/cluster.py

```diff
@@ -318,10 +318,6 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False, wait_
                     if old_node is not node:
                         old_node.watch_log_for_alive(node, from_mark=mark)
 
-        if wait_for_binary_proto:
```

Comment:
This worries me, and I think I'd like to undo this change, assuming DSE still works w/o it.